.github/workflows/delete-deployment.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: Delete deployment
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+    types:
+      - unlabeled
```

Comment:
A lot of the code for deployments is common, please, consider using more reusable workflows: https://github.blog/2022-02-10-using-reusable-workflows-github-actions/

Reply:
I definitely will use that in next PR, but for now I think that we can work out with this setup.